site.yml

```diff
@@ -9,20 +9,21 @@ content:
     - HEAD
   - url: https://github.com/owncloud/docs.git
     branches:
-    - '10.6'
-  - url: https://github.com/owncloud/android.git
+    - '10.8'
+    - '10.7'
```

Comment:
This looks like a fix for an existing out-of-date branches list.

Reply:
thanks for catching 🤦‍♂️ 
fixed